features/admin.claims.v1/components/edit/external-dialect/edit-external-claims.tsx

```diff
@@ -159,6 +167,43 @@ export const EditExternalClaims: FunctionComponent<EditExternalClaimsPropsInterf
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
 
+    useEffect(() => {
+        if (!attributeUri || !dialectID) {
+            return;
+        }
+        if (!SCIMConfigs.serverSupportedClaimsAvailable.includes(attributeUri)) {
+            return;
+        }
+        getServerSupportedClaimsForSchema(dialectID)
+            .then((response: ServerSupportedClaimsInterface) => {
+                setServerSupportedClaims(response.attributes);
+            })
+            .catch((error: IdentityAppsError) => {
```

Comment:
Use `IdentityAppsApiException` instead of this.

Reply:
As per the offline discussion we had, `IdentityAppsError` seems to match the properties returned by the API error. So we can keep this.